packages/mongodb-chatbot-ui/src/FloatingActionButtonTrigger.tsx

```diff
@@ -29,8 +29,8 @@ export function FloatingActionButtonTrigger(
   props: FloatingActionButtonTriggerProps
 ) {
   const { darkMode } = useDarkMode(props.darkMode);
-  const { openChat } = useChatbotContext();
-  const text = props.text ?? "MongoDB AI";
+  const { chatbotName, openChat } = useChatbotContext();
+  const text = props.text ?? `Ask ${chatbotName}`;
```

Comment:
i believe `chatbotName` can be `undefined`. in which case if `props.txt` is `undefined` and `chatbotName` is undefined, this'd be "Ask " or "Ask undefined" (not sure behavior here).

if chatbotName can be undefined here, maybe the backup could be text is an empty string or a generic phrase like "Ask the chatbot"

Reply:
Oops yeah good catch!